src/main/java/org/broadinstitute/consent/http/models/DarCollectionSummary.java

```diff
@@ -19,6 +19,9 @@ public class DarCollectionSummary {
   @JsonProperty
   private Integer darCollectionId;
 
+  @JsonProperty
+  private Set<String> referenceIds;
```

Comment:
Optional: I generally prefer `List` over `Set` simply because they're easier to work with. Certainly not a blocker if you feel that `Set` is a better fit here.

Reply:
I think, given the context of the query, the Set makes sense. The joins on elections and votes will have referenceId repeated on multiple rows. With a Set the `DARCollectionSummaryReducer` can add the value without worrying about having duplicates.